Library/TestHelpers/XCTestCase+AppEnvironment.swift

```diff
@@ -4,6 +4,7 @@ import KsApi
 import ReactiveSwift
 import XCTest
 
+// swiftlint:disable line_length
 extension XCTestCase {
   // Pushes an environment onto the stack, executes a closure, and then pops the environment from the stack.
   func withEnvironment(_ env: Environment, body: () -> Void) {
```

Comment:
When adding stuff to `Environment` be sure to also carry the arguments through to the `pushEnvironment` and `replaceCurrentEnvironment` functions in `Library/AppEnvironment.swift` too 👍